docs/services/passbolt.md

```diff
@@ -0,0 +1,123 @@
+# passbolt
+
+Passbolt ist ein self-hosted open source Passwortmanager, welcher sehr gut für Teams geeignet ist, durch die Funktion, 
+dass man intern Passwörter teilen kann und Teams erstellen kann.
+
+```yaml
+version: '3.9'
+
+services:
+  postgres:
+    image: postgres
+    restart: always
+    env_file: .postgres.env
+    environment:
+      - "POSTGRES_DB=passbolt"
+    volumes:
+      - "/srv/passbolt/postgres:/var/lib/postgresql/data"
+
+  passbolt:
+    image: passbolt/passbolt:latest
+    restart: always
+    tty: true
+    depends_on:
+      - postgres
+    env_file: .passbolt.env
+    environment:
```

Comment:
please add ports, so it's ready to be used with nginx
```yaml
  ports:
    - "[::1]:8000:80"
```

Reply:
The Ports are in the nginx section. Same structure like in paperless

Reply:
Nope ports are defined in both sections. See https://github.com/felbinger/AdminGuide/blob/main/docs/services/paperless.md?plain=1#L45-L46 for reference